modules/support-gather-data.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="support_gathering_data_{context}"]
+= Gathering data about your cluster for Red Hat Support
+
+When opening a support case, it is often helpful to provide debugging
+information about your cluster to Red Hat Support. You can gather this
+information by using the `oc adm must-gather` CLI command.
+
+The `oc adm must-gather` CLI command collects the information from your cluster
+that is most likely needed for debugging issues, such as:
+
+* Resource definitions
+* Audit logs
+* Service logs
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Run the `oc adm must-gather` command:
++
+----
+$ oc adm must-gather
+----
++
+This creates a Pod on the cluster to perform the data collection on. All of the
+collected resources and data from your cluster are saved in the current
+directory, in a new directory that starts with `must-gather.local`.
+
+. Create a compressed file from the `must-gather` directory. For example, on a
+computer that uses a Linux operating system, run the following commands:
++
+----
+$ tar cvf must-gather.tar must-gather.local.5421342344627712289/
```

Comment:
Drop the gzip command (you don't need it - tar can do this for you). 

```
tar cvaf must-gather.tar.xz must-gather.local.5421342344627712289/
```

- in short `-a` with tar is the 'auto' (for compression) option and it will use the file extension you provide to determine how to compress your archive. In my example, using the `xz` compression method (however `.gz` would use the `gzip` compression method)

Reply:
Good deal, I can make this update, thanks!